docs/troubleshooting.md

```diff
@@ -62,3 +63,79 @@ For instance, try this:
 # oc create --raw /apis/admission.hive.openshift.io/v1/dnszones -f config/samples/hiveadmission-review-failure.json -v 8 | jq
 ```
 
+## Regaining access to a cluster after changing the API certificate
+
+There are several ways to regain access to a cluster after its API certificate has been changed.  Usually this will manifest as an error message similar to: `x509: certificate signed by unknown authority`.
```

Comment:
I think we should frame this from the perspective of having a `ClusterDeployment` in this state with an `Unreachable` condition containing the `x509: certificate signed by unknown authority` error message.